src/MSCorEE.Desktop/IClrMetaHost.cs

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+    using System.Runtime.CompilerServices;
+    using System.Runtime.InteropServices;
+    using System.Text;
+
+    /// <summary>
+    /// Provides methods that return a specific version of the common language runtime (CLR) based on its version number, list all installed CLRs, list all runtimes that are loaded in a specified process, discover the CLR version used to compile an assembly, exit a process with a clean runtime shutdown, and query legacy API binding.
+    /// </summary>
+    /// <code>
+    /// ICLRMetaHost* pMetaHost = NULL;
+    /// HRESULT hr = CLRCreateInstance(CLSID_CLRMetaHost, IID_ICLRMetaHost, (LPVOID*) &amp; pMetaHost);
+    /// </code>
+    /// <remarks>
+    /// .NET Framework Versions: 4.5, 4.
+    /// </remarks>
+    [InterfaceType(ComInterfaceType.InterfaceIsIUnknown)]
+    [Guid("D332DB9E-B9B3-4125-8207-A14884F53216")]
+    [ComImport]
+    public interface IClrMetaHost
```

Comment:
If you lifted this interface definition from someplace, can you add a notice (perhaps to the xml doc comments) providing the author's name and/or the URL?